config/initializers/new_framework_defaults_6_1.rb

```diff
@@ -13,11 +13,11 @@
 # Rails.application.config.active_storage.track_variants = true
 
 # Apply random variation to the delay when retrying failed jobs.
-# Rails.application.config.active_job.retry_jitter = 0.15
+Rails.application.config.active_job.retry_jitter = 0.15
```

Comment:
This got me looking up about the thundering herd problem. TIL

Reply:
I know, but marking the default as active means less mental overhead when we eventually get rid of this file and try to see what can just be removed and what needs to move as an override into `application.rb` 😃